study.md

```diff
@@ -27,17 +27,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+The DOM is not HTML
+The DOM is not a programming language.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes. The Dev Tools create the DOM using CSS selectors
```

Comment:
The dev tools is not the actual DOM but more of a visual representation of the DOM, to be pedantic :-P